site/pages/change-password.test.tsx

```diff
@@ -0,0 +1,56 @@
+import renderer from "react-test-renderer";
+import { describe, it, expect, vi, beforeEach } from "vitest";
+import ChangePassword, { ChangePasswordPageProps } from "./change-password.page";
+
+const blankInputs: ChangePasswordPageProps = {
+    errors: [],
+    inputs: {},
+};
+
+const withInputsAndNoErrors: ChangePasswordPageProps = {
+    errors: [],
+    inputs: {
+        currentPassword: "dummyPassword",
+        newPassword: "dummyPassword",
+        confirmPassword: "dummyPassword",
+    },
+};
+
+const withInputsAndErrors: ChangePasswordPageProps = {
+    ...withInputsAndNoErrors,
+    errors: [{ errorMessage: "Enter a valid email address", id: "currentPassword" }],
+};
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
We dont need to use requires. If you search the repo for ' vi.spyOn ' you can find examples.